Core/include/Acts/Seeding/SeedFinderOrthogonal.ipp

```diff
@@ -226,6 +226,18 @@ bool SeedFinderOrthogonal<external_spacepoint_t>::validTuple(
     return false;
   }
 
+  /*
+   * Cut: Ensure that inner SPs is in a certain (r, eta) region of the detector
+   * for fast seeding.
+   */
+  if (m_config.fastTrackingCut) {
+    const float rInner = (isMiddleInverted) ? rH : rL;
+    if (rInner < m_config.fastTrackingRMin and
+        std::fabs(cotTheta) > m_config.fastTrackingCotThetaMax) {
```

Comment:
which one of these condition happens more frequently? put it first

Reply:
yes, I believe `rInner < m_config.fastTrackingRMin` should go first